APIAlfresco.php

```diff
@@ -568,4 +589,24 @@ private function extractSpecialCharacters($word)
 
         return $word;
     }
+
+    private function processException($e)
```

Comment:
If you are writing a library, you don't need to display any error using (echo, log, stdout, etc).
That's something the client-developer should handle it in the **catch** statement.
I'm not sure if this function is really helpful.